src/handlers/proxy.ts

```diff
@@ -0,0 +1,25 @@
+import { Context, Env, SupportedEventsU } from "../types";
+import { userSelfAssign, userStartStop } from "./user-start-stop";
+
+export interface Result {
+  status: "ok" | "failed" | "skipped";
```

Comment:
If there are only three statuses maybe the HTTP status code idea is overkill. What do you think makes sense @gentlementlegen ? I assumed this was some type of plugin standard.

Reply:
It's not standardized as these types should actually come from the Kernel. I don't remember if we finally exported these, I will have a look.